### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Generated by https://github.com/jsoref/spelling `f`; to maintain your repo, please consider `fchurn`

### CODE REVIEW

File: modules/cli/src/main/resources/completions/zsh [hunk 1]
```diff
@@ -49,7 +49,7 @@ _coursier() {
   local property=( {--property,-D}'[set Java properties in the generated launcher]:key-value:' )
   local quiet=( {--quiet,-q}'[quiet output]' )
   local repository=( {--repository,-r}'[repository; for multiple repositories, separate with comma and/or add this option multiple times]:repository:' )
-  local reverseTree=( {--reverse-tree,-T}'[print dependencies as an inversed tree; dependees as children]' )
+  local reverseTree=( {--reverse-tree,-T}'[print dependencies as an inverted tree; dependees as children]' )
```
Comment (jsoref) [inline modules/cli/src/main/resources/completions/zsh]: I'm not terribly confident about `inverted`. `inversed` isn't a word, `reversed` would be the natural word to use based on the flag `--reverse-tree`.

File: modules/web/src/main/resources/index.html [hunk 2]
```diff
@@ -89,7 +89,7 @@
 
 <script>
 // From http://www.bootply.com/VQqzOawoYc
-// Javascript to enable link to tab
+// JavaScript to enable link to tab
```
Comment (jsoref) [inline modules/web/src/main/resources/index.html]: If this is actually from someone else's code, I could understand not applying it.

I tend to eventually chase down upstreams if people provide pointers.

File: modules/cli/src/main/resources/completions/zsh (hunk 1 above)
Comment (alexarchambault) [reply modules/cli/src/main/resources/completions/zsh]: `reversed` would be fine, yes.
```suggestion
  local reverseTree=( {--reverse-tree,-T}'[print dependencies as a reversed tree; dependees as children]' )
```

File: modules/cli/src/main/scala/coursier/cli/deprecated/CommonOptions.scala [hunk 3]
```diff
@@ -13,7 +13,7 @@ final case class CommonOptions(
   @Help("Print dependencies as a tree")
   @Short("t")
     tree: Boolean = false,
-  @Help("Print dependencies as an inversed tree (dependees as children)")
+  @Help("Print dependencies as an inverted tree (dependees as children)")
```
Comment (alexarchambault) [inline modules/cli/src/main/scala/coursier/cli/deprecated/CommonOptions.scala]:
```suggestion
  @Help("Print dependencies as a reversed tree (dependees as children)")
```

File: modules/cli/src/main/scala/coursier/cli/resolve/ResolveOptions.scala [hunk 4]
```diff
@@ -17,7 +17,7 @@ final case class ResolveOptions(
   @Help("Print dependencies as a tree")
   @Short("t")
     tree: Boolean = false,
-  @Help("Print dependencies as an inversed tree (dependees as children)")
+  @Help("Print dependencies as an inverted tree (dependees as children)")
```
Comment (alexarchambault) [inline modules/cli/src/main/scala/coursier/cli/resolve/ResolveOptions.scala]:
```suggestion
  @Help("Print dependencies as a reversed tree (dependees as children)")
```

File: modules/web/src/main/resources/index.html (hunk 2 above)
Comment (alexarchambault) [reply modules/web/src/main/resources/index.html]: This was initially generated by docusaurus I think. It's fine fixing here.

**Comment by alexarchambault** [comment]
Merging, thanks @jsoref!

**Pull request merged**